### PR TITLE
Update blizz from 15.11.4 to 15.12.6 and rename to teamviewermeeting

### PR DESCRIPTION
**Pull request by suschizu**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [ ] There are no [open pull requests](https://github.com/Homebrew/homebrew-cask/pulls) for the same update.
- [ ] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Created with [`cask-repair`](https://github.com/Homebrew/homebrew-cask/blob/master/CONTRIBUTING.md#updating-a-cask).

### CODE REVIEW

File: Casks/blizz.rb [hunk 1]
```diff
@@ -1,12 +1,12 @@
 cask "blizz" do
```
Comment (reitermarkus) [inline Casks/blizz.rb:1]: This should be renamed.

**Comment by suschizu** [comment]
@reitermarkus - I tried to but recieved this error

`/home/linuxbrew/.linuxbrew/Homebrew/Library/Homebrew/cask/cask_loader.rb:85:in `cask': Cask 'blizz' definition is invalid: Token 'teamviewermeeting' in header line does not match the file name. (Cask::CaskTokenMismatchError)`

 perhaps change the name after merging?

**Comment by reitermarkus** [comment]
The file name also had to be changed.

**Comment by suschizu** [comment]
sure 🤦‍♀️

**Review by miccal — APPROVED** [review]

**Pull request merged**